services/src/test-library/src/main/java/org/openkilda/testing/Constants.java

```diff
@@ -27,6 +29,7 @@
     public static final Integer RULES_INSTALLATION_TIME = 5;
     public static final Integer STATS_LOGGING_TIMEOUT = 70;
     public static final SwitchId NON_EXISTENT_SWITCH_ID = new SwitchId("de:ad:be:ef:de:ad:be:ef");
+    public static final String NON_EXISTENT_FLOW_ID = "fake " + UUID.randomUUID().toString();
```

Comment:
I vote that 'non-existent' would be in the name. In the future, it will help us to handle logs.

Reply:
Yes, `static final` should definitely not be generated different every time. I would understand if this was some kind of `NON_EXISTENT_FLOW_ID_PREFIX` and then you would add a random part every time you call it in the test, but would still find it as an overkill.